src/dataaccess/HardCodedSarcomaPatient.json

```diff
@@ -1087,658 +938,290 @@
-                            "EntryType": {
-                                "Value": "http://standardhealthrecord.org/spec/shr/core/Coding"
-                            },
-                            "Value": "6064005",
-                            "CodeSystem": {
-                                "EntryType": {
-                                    "Value": "http://standardhealthrecord.org/spec/shr/core/CodeSystem"
-                                },
-                                "Value": "http://snomed.info/sct"
-                            },
-                            "DisplayText": {
-                                "EntryType": {
-                                    "Value": "http://standardhealthrecord.org/spec/shr/core/DisplayText"
-                                },
-                                "Value": "Topical route"
-                            }
-                        }
-                    ]
+                    "Value": "http://standardhealthrecord.org/spec/shr/core/CodeableConcept"
                 }
             }
-        }
-    },
-    {
-        "EntryType": {
-            "Value": "http://standardhealthrecord.org/spec/shr/medication/MedicationRequested"
         },
-        "ShrId": "788dcbc3-ed18-470c-89ef-35ff91854c7f",
-        "EntryId": "18",
         "PersonOfRecord": {
-            "_ShrId": "788dcbc3-ed18-470c-89ef-35ff91854c7f",
+            "_ShrId": "788dcbc3-ed18-470c-89ef-35ff91854c7e",
             "_EntryId": "1",
             "_EntryType": {
                 "Value": "http://standardhealthrecord.org/spec/shr/entity/Patient"
             }
         },
-        "CreationTime": {
-            "EntryType": {
-                "Value": "http://standardhealthrecord.org/spec/shr/core/CreationTime"
-            },
-            "Value": "15 FEB 2018"
-        },
-        "LastUpdated": {
-            "EntryType": {
-                "Value": "http://standardhealthrecord.org/spec/shr/base/LastUpdated"
-            },
-            "Value": "15 FEB 2018"
-        },
-        "Author": {
-            "EntryType": {
-                "Value": "http://standardhealthrecord.org/spec/shr/base/Author"
-            },
-            "Value": "Dr. ABC123"
-        },
         "ActionContext": {
+            "Status": {
+                "Value": "completed",
+                "EntryType": {
+                    "Value": "http://standardhealthrecord.org/spec/shr/action/Status"
+                }
+            },
             "EntryType": {
                 "Value": "http://standardhealthrecord.org/spec/shr/action/RequestedContext"
             },
             "Reason": [
                 {
-                    "EntryType": {
-                        "Value": "http://standardhealthrecord.org/spec/shr/core/Reason"
-                    },
                     "Value": {
-                        "_ShrId": "788dcbc3-ed18-470c-89ef-35ff91854c7f",
+                        "_ShrId": "788dcbc3-ed18-470c-89ef-35ff91854c7e",
```

Comment:
[LC1] Some of the _ShrId end with an e and some end with an f. Want to makre sure that everything has the same shr id.